libretroshare/src/retroshare/rsflags.h

```diff
@@ -36,14 +36,21 @@ using rs_is_scoped_enum = std::integral_constant< bool,
  * it as flags type passing it as parameter of this macro.
  * The result will be type safe flags, that cannot be mixed up with flag of a
  * different type, but that are very comfortable to operate like plain old
- * integers.
+ * integers. All commom operation like &, | or ! can be used. To convert
+ * the result of such operation use !!:
```

Comment:
Better being a bit more explicit, say at least to convert to bool ;)

Reply:
Thanks :grinning: 
fixed and force pushed..